application/models/Traits/PermissionTrait.php

```diff
@@ -31,6 +31,29 @@ public static function getMinimalPermissionRead()
         return null;
     }
 
+    /**
+     * get criteria from Permission
+     * @todo : create an event
+     * @param integer $userid
+     * @return CDbCriteria
+     */
+    public static function getPermissionCriteria(/** @scrutinizer ignore-unused */ $userid = null)
+    {
+        $criteriaPerm = new CDbCriteria();
+        return $criteriaPerm;
+    }
+
+    /**
+     * Scope for permission
+     * @param integer $userid
+     * @return self
+     */
+    public function withpermission($userid = null)
```

Comment:
I believe is important to get to a proper name to avoid missusing or miss interpretation.

Reply:
1. withListAccess
2. withMinimumRight (Right because != Permission ;) )

Reply:
I take both ;)  withListRight

Ok ?